pyexr/exr.py

```diff
@@ -34,7 +34,7 @@ def open(filename):
   return InputFile(OpenEXR.InputFile(filename), filename)
 
 
-def read(filename, channels = "default", precision = FLOAT):
+def read(filename, channels = "all", precision = FLOAT):
```

Comment:
This is indeed what the README says, but this change would break backward compatibility. I would rather keep the default to 'default'. Do you think 'all' would be a better default?

Reply:
Hmm... I think backward compatibility is more important.

The default `channels` of `read()` and `InputFile.get()` can be both `default`, and an example of `read_all()` and `InputFile.get_all()` can be added into `README.md`.

Reply:
That sounds good.